internal/transfertask/config.go

```diff
@@ -20,10 +20,17 @@ type GlobusTransferConfig struct {
 	DestinationTemplate     string   `yaml:"destinationTemplate"`
 }
 
+type ExtGlobusTransferConfig struct {
+	TransferServiceUrl string `string:"TransferServiceUrl" validate:"http_url"`
+	SrcFacility        string `string:"SrcFacility" validate:"required"`
+	DstFacility        string `string:"DstFacility" validate:"required"`
```

Comment:
```suggestion
	DstFacility        string `string:"DestinationFacility" validate:"required"`
```
User facing config should be more readable imho.

Reply:
I guess that means SrcFacility should also be "SourceFacility". I'll change that.